### ISSUE TITLE
Add a simple machine learning algorithm on the streaming temperature data.

### ISSUE THREAD

**Issue by shiv4nsh** [shiv4nsh/raspi-spark-streaming-mqtt#1]

**Comment by tspannhw** [comment]
what kind of algorithm?   what are you thinking will be of value?  predict future temp?

**Comment by shiv4nsh** [comment]
I am open to suggestions ! Please suggest something

**Comment by tspannhw** [comment]
> I am open to suggestions ! Please suggest something

Just saw your article.
I am thinking this:
  https://github.com/arnesund/weather-ml

The CPU temperature of Pis is not a great sensor.

https://www.raspberrypi.org/products/sense-hat/  is cheap and gives you a few more sensors.

**Comment by shiv4nsh** [comment]
Yeah sure , we can go with sense hat , btw its not that much cheap costs 25 euro that's around 2200 bucks in India ! ;) But yes we can go with that

**Comment by aj0408** [comment]
It will be interesting to go with multiple features for a machine learning example. e.g. the features could be
1. CPU usage
2. temperature
3. file IO
4. network IO
etc. 

Now create a model to see if the temperature is anomalous, based on all the feature streams.

**Comment by tspannhw** [comment]
those sounds like good ideas
